00-Lab/Introduction.ipynb

```diff
@@ -143,15 +137,99 @@
+    "\n",
+    "Several people independently working on the same file/dataset is a recipe for confusion if you don't have some type of version control management. Enter Git. This resource, once understood, will be your best ally in this mad world of data. In it's simplist form, Git is a way of keeping track of changes you make to your files, so you can go back in time in case something went wrong or if you wanted to take your data/scripts/files in another direction. Please read the [Git Parable](http://tom.preston-werner.com/2009/05/19/the-git-parable.html) to understand how git is used in the software development world, but draw connections with your own work and how the same concepts can apply.\n",
+    "\n",
+    "### [Sign up for an account on github](https://github.com/) (if you haven't already)\n",
+    "This account will be your portal to the open access world of software and sharing, try to use your hawkid if available.\n",
+    "\n",
+    "### Send us your username on slack! (if we haven't added you to the class already)\n",
+    "Please send us your username on slack so we can find you in the github universe and add you to the course.\n",
+    "cl\n",
+    "### Set-up git in your virtual machine\n",
+    "**[Supplemental Video](https://www.lynda.com/Git-tutorials/Configuring-Git/100222/111256-4.html) (login to lynda with your hawkid to get access to the full course)**\n",
+    "\n",
+    "Inside the virtual machine, click on the little computer at the bottom of your screen and a black box with a flashing cursor will appear. This is your terminal, and it can be a powerful tool to automate menial tasks, but right now, we are going to use it to set up our git account.\n",
+    "\n",
+    "The first step is set up our username by following the template command below in your terminal, and hit **ENTER** once you are done typing.\n",
+    "```bash\n",
+    "git config --global user.name \"<hawid> or whatever your github account name is\"\n",
+    "```\n",
+    "**EXAMPLE**\n",
+    "```bash\n",
+    "git config --global user.name \"jdkent\"\n",
+    "```\n",
+    "\n",
+    "The next step is to set up our email using the command template below and hit **ENTER** once you are done typing:\n",
+    "```bash\n",
+    "git config --global user.email \"email@uiowa.edu\"\n",
+    "```\n",
+    "**EXAMPLE**\n",
+    "```bash\n",
+    "git config --global user.email \"james-kent@uiowa.edu\"\n",
+    "```\n",
+    "\n",
+    "Now you can check to make sure you did everything correctly by typing the following\n",
+    "```bash\n",
+    "git config --list\n",
+    "```\n",
+    "\n",
+    "\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Step 6: Further Reading/Resources\n",
+    "## Step 7: Getting your class repository\n",
```

Comment:
I want pictures or a video to go here to aide in the explanation, and perhaps links to better explain the jargon.